f3/granite.go

```diff
@@ -4,18 +4,24 @@ import (
 	"bytes"
 	"encoding/binary"
 	"fmt"
-	"sort"
-
 	"github.com/filecoin-project/go-bitfield"
 	rlepluslazy "github.com/filecoin-project/go-bitfield/rle"
 	"golang.org/x/xerrors"
+	"math"
+	"sort"
 )
 
 type GraniteConfig struct {
 	// Initial delay for partial synchrony.
 	Delta float64
 	// Change to delta in each round after the first.
 	DeltaRate float64
+	// Absolute extra value to add to delta after the first few unsuccessful rounds.
+	DeltaExtra float64
+	// Time to next clock tick (according to local clock)
+	// Used in timeouts in order to synchronize participants
+	// (by not carrying significant delays from previous/starting rounds)
+	ClockTickDelta float64
```

Comment:
"Clock tick" is a very confusing name, it took me forever to work out. I think you mean something more like epoch duration (when it referred to drand). We're not using drand though, so it's more just an external clock re-sync point. I suggest a name like `ExternalClockResyncPeriod`. Update comments accordingly.

Reply:
Done.